src/components/sidebar/components/sidebar-title-heading/index.tsx

```diff
@@ -0,0 +1,16 @@
+import sSidebar from 'components/sidebar/sidebar.module.css'
```

Comment:
Nit, non-blocking: what if we co-located `SidebarTitleHeading` & `SidebarSectionHeading` in the same file? Potential benefits:

- hierarchical relationship between the two might be clearer
- we'd have `sidebar-title-heading.module.css` which could have one className rather than separate but identical `.title` & `.heading`; this might also make it more clear that the two components share identical styles

Reply:
I really like this idea! Could also lower the risk for misusing one for the other. I'll work on this update. 😊